internal/distro/rhel90/distro.go

```diff
@@ -552,7 +574,8 @@ func newDistro(distroName string) distro.Distro {
 
 	// Shared Services
 	edgeServices := []string{
-		"NetworkManager.service", "firewalld.service", "sshd.service",
+		// TODO(runcom): move fdo-client-linuxapp.service to presets?
+		"NetworkManager.service", "firewalld.service", "sshd.service", "fdo-client-linuxapp.service",
```

Comment:
we'd also need an onboarding test plan like:

- configure and bring up a rendezvous and onboarding server
- configure and bring up a manufacturing server
- configure SSH in the onboarding server server info
- install
-  verify the ssh is there